pkg/pool-manager/mac-pool-map.go

```diff
@@ -0,0 +1,118 @@
+package pool_manager
```

Comment:
rename to `pkg/pool-manager/macpoolmap.go`

Reply:
Done